packages/itwinui-css/src/utils/mixins.scss

```diff
@@ -12,7 +12,7 @@
   }
 }
 
-@mixin iui-focus($color: var(--iui-color-foreground-primary), $offset: -1px, $thickness: 1px) {
+@mixin iui-focus($color: var(--iui-color-text-primary), $offset: -1px, $thickness: 1px) {
```

Comment:
I think this should use `border-primary`. They refer to same color underneath but border makes more sense

Reply:
Agreed, change made.